qbft/instance.go

```diff
@@ -90,6 +90,10 @@ func (i *Instance) Broadcast(msg *SignedMessage) error {
 
 // ProcessMsg processes a new QBFT msg, returns non nil error on msg processing error
 func (i *Instance) ProcessMsg(msg *SignedMessage) (decided bool, decidedValue []byte, aggregatedCommit *SignedMessage, err error) {
+	if i.State.Round == CutoffRound {
+		return false, nil, nil, errors.New("round > cutoff round")
+	}
```

Comment:
Didn't see this is the SIP... only the `timeout` section was changed.
Doesn't it make better sense to finish the `runner` instead?

In any case just to be safer:
```suggestion
	if i.State.Round >= CutoffRound {
		return false, nil, nil, errors.Errorf("round %d > cutoff round %d", i.State.Round, CutoffRound)
	}
```

Reply:
This is the instance, not the runner